src/gui/rss/automatedrssdownloader.cpp

```diff
@@ -315,9 +311,6 @@ void AutomatedRssDownloader::updateRuleDefinitionBox(QListWidgetItem *selected)
             else
                 lMatch = tr("Last Match: Unknown");
             ui->lblLastMatch->setText(lMatch);
-            updateMustLineValidity();
-            updateMustNotLineValidity();
-            updateEpisodeFilterValidity();
```

Comment:
Not anymore, that's why I changed the signals used with the validity checks to use textChanged. textChanged will fire when changing the text programmatically, where as textEdited only fires when changed by user input. So when the selection changes, the validity checks will fire when the new values are populated